mkosi/resources/man/mkosi.1.md

```diff
@@ -2036,6 +2036,7 @@ There are also specifiers that are independent of settings:
 | `%C`      | Parent directory of current config file |
 | `%P`      | Current working directory               |
 | `%D`      | Directory that mkosi was invoked in     |
+| `%I`      | The name of the current subimage        |
```

Comment:
Shouldn't that go in the table above?

Reply:
I wouldn't say so, as `config.image` isn't controllable via a setting